tests/run-make/print-native-static-libs/rmake.rs

```diff
@@ -0,0 +1,56 @@
+//! This checks the output of `--print=native-static-libs`
```

Comment:
Could you elaborate a bit on the context: what does this test intend to test and why having consecutively present args is problematic? (As in, yes I can see it's trying to examine the output of `--print=native-static-libs` but what is it *attempting* to test?)

Reply:
Sure, extended the comment with more context as well as more details on the test itself.

But to answer your question directly, the issue with duplicated constitutive args is mainly that they are useless and they clog the output. I added a link the issue in the comment.

Reply:
Ah thanks, I suspected as much but did not know the context for certain.